c/include/libsbp/observation.h

```diff
@@ -340,39 +340,75 @@ typedef struct SBP_ATTR_PACKED {
 } msg_ephemeris_qzss_t;
 
 
```

Comment:
I'm sure I've done something against protocol in this PR. :)

Reply:
Only a generated header file is changed here - did you change that by hand? Look at past PR's around new deprecated messages for guidance here.